CodeEditModules/Modules/CodeFile/src/TextView/CodeEditorTextView.swift

```diff
@@ -89,4 +89,156 @@ public class CodeEditorTextView: NSTextView {
             replacementRange: selectedRange()
         )
     }
+
+    /// input a backslash (\\)
+    @IBAction func inputBackSlash(_ sender: Any?) {
+        super.insertText(
+            "\\",
+            replacementRange: selectedRange()
+        )
+    }
+
+    /// Override of the default context menu in the editor...
+    ///
+    /// This is a basic view without any functionality, once we have most the items built
+    /// we will start connecting the menu items to their respective actions.
+    public override func menu(for event: NSEvent) -> NSMenu? {
+        guard var menu = super.menu(for: event) else { return nil }
+
+        menu = helpMenu(menu)
+        menu = codeMenu(menu)
+        menu = gitMenu(menu)
+        menu = removeSubMenus(menu)
+        menu = removeTextMenu(menu)
+
+        return menu
+    }
+
+    func helpMenu(_ menu: NSMenu) -> NSMenu {
+        menu.insertItem(withTitle: "Jump To Definition",
+                        action: nil,
+                        keyEquivalent: "",
+                        at: 1)
+
+        menu.insertItem(withTitle: "Show Code Actions",
+                        action: nil,
+                        keyEquivalent: "",
+                        at: 2)
+
+        menu.insertItem(withTitle: "Show Quick Help",
+                        action: nil,
+                        keyEquivalent: "",
+                        at: 3)
+
+        menu.insertItem(.separator(), at: 4)
+
+        return menu
+    }
+
+    func codeMenu(_ menu: NSMenu) -> NSMenu {
+
+        menu.insertItem(withTitle: "Refactor",
+                        action: nil,
+                        keyEquivalent: "",
+                        at: 5)
+
+        menu.insertItem(withTitle: "Find",
+                        action: nil,
+                        keyEquivalent: "",
+                        at: 6)
+
+        menu.insertItem(withTitle: "Navigate",
```

Comment:
Please make it localized.
add `.localized()` and add it to `Localizable.strings`